docs/release-notes/RELEASE-NOTES-3.2.0.md

```diff
@@ -76,13 +80,13 @@ Here are example nodes that can be used to inspect current top block and see inf
 
 ## Join the testnet
 
-This section describes how to run a node as part of the testnet - the public test network of nodes - by using the release binary.
+In order to join the testnet change the [Network ID](../../docs/configuration.md#network-id) in node configuration file to `ae_uat`.
```

Comment:
Can we avoid this link? Considering that some doc files are included in the resulting package (see rebar.config, and the resulting tarball), the reader of this doc in the untarred package would be confused.

Reply:
Ignore ^^^ - it seems we already use this approach. Checking.

Reply:
Seems fine based on rebar.config

Reply:
There is already links to other docs like: `[the dedicated separate document](../../docs/installation.md).`

Reply:
Non linking harms the online experience, and I would trade of the offline (package) experience for online as I doubt someone read docs offline (in the package).